### ISSUE TITLE
🐞 [Bug]: Deployment is stuck at validating Nodes on the Dashboard, and the old deployment lists are not visible.

### ISSUE THREAD

**Issue by mahendravarmayadala93** [threefoldtech/tfgrid-sdk-ts#3173]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### which package/s did you face the problem with?

Dashboard

### What happened?

Mat is unable to connect to a VM on Node ID 687 due to the deployed list is not visible on his dashboard which he deployed 6 months back. Also, check the specific [issue ](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3244)for this showing logs..  This issue follows a previous [issue ](https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2435)where deployments were also not displayed.

He attempted to deploy a new VM on Node ID 687, but the process is stuck on node validation.

A [testing video](https://drive.google.com/file/d/1F8_fqVOzyzI5gqSsvCdlkVaHQUQlek8Y/view?usp=drive_link) is available for reference from our side where In contrast Scott , Sherwin, Shang, and I successfully deployed on Node ID 687 without any issues, and the process went smoothly.

Also, here are the errors on the client side on his browser console while trying to deploy on his Node : 687

![5848377467574797704](https://github.com/user-attachments/assets/ef0f8f5d-28a7-40f6-aba2-a4f5129f861c)

We tested the same between Node 687 & Node 1, you can see the difference in errors

![image](https://github.com/user-attachments/assets/4777b180-86e0-4141-bbd6-ccdbec489c0e)

![6143390348706169709](https://github.com/user-attachments/assets/0f5f6cc5-9e61-48c2-afd2-da8e41ce3657)

### Steps To Reproduce

1) Open Dashboard

2) After you login click on Side menu, and click on Deploy and then choose Virtual machines

3) Choose any one of the VM

4) Select Nodes manually and then choose the Node 687 

5) Then click on deploy.

### which network/s did you face the problem on?

Main

### version

NA

### Twin ID/s

NA

### Node ID/s

687

### Farm ID/s

209

### Contract ID/s

NA

### Relevant screenshots/screen records

Here's the screen record on Client side while deploying VM on the Node 687 : https://drive.google.com/file/d/1rcWeU59E6SXyKpFc88SdB9Fmnd1om31u/view?usp=sharing

### Relevant log output

```console
NA
```

**Comment by TullysInc** [comment]
Update from the farmer: https://t.me/threefoldtesting/15012

He is now unable to deploy a VM on any node in Poland.

![image](https://github.com/user-attachments/assets/5ad0832e-4fcf-4093-8c3f-ee35cdc77326)

**Comment by Cameron1986labeyrie** [comment]
Update from Farmer on Crisp:

- The user is still unable to deploy on his own node_687.
- Strangely the issue is occuring with any node he chooses in Poland.

![687-v2_gnkkts](https://github.com/user-attachments/assets/cf604acf-8f7e-499a-91fe-a8d813d0d508)

![687-v1_45dngt](https://github.com/user-attachments/assets/f2849395-76db-4286-9402-52995a46c807)

**Comment by scottyeager** [comment]
The user reports that they are able to create deployments after creating a new account.

I suspect that the issue may be with the RMB public key set on tfchain for the original account.

**Comment by AhmedHanafy725** [comment]
> I suspect that the issue may be with the RMB public key set on tfchain for the original account.

sound yes. but the dashboard auto-updates the public key once login if it's incorrect. so what is the twin ID of the old account?

**Comment by scottyeager** [comment]
Right, I would have expected the public key to get updated too, especially since the user logged in again using incognito tabs and also tried multiple browsers.

The twin id is 737.

**Comment by AhmedHanafy725** [comment]
could be this cache issue as explained here https://github.com/threefoldtech/rmb-rs/issues/200#issuecomment-2435549736

**Comment by TullysInc** [comment]
> could be this cache issue as explained here [threefoldtech/rmb-rs#200 (comment)](https://github.com/threefoldtech/rmb-rs/issues/200#issuecomment-2435549736)

@AhmedHanafy725  - I see the corresponding ticket is closed. Can you confirm if this issue is resolved?

**Comment by AhmedHanafy725** [comment]
> [@AhmedHanafy725](https://github.com/AhmedHanafy725) - I see the corresponding ticket is closed. Can you confirm if this issue is resolved?

it should be yes

**Comment by Cameron1986labeyrie** [comment]
> > [@AhmedHanafy725](https://github.com/AhmedHanafy725) - I see the corresponding ticket is closed. Can you confirm if this issue is resolved?
> 
> it should be yes

Thank you for the update @AhmedHanafy725

**Comment by atefnazmy** [comment]
- Verified on Mainnet,
- Version:  2.7.0
- The VM was successfully deployed and listed on a node in Poland without any issues.
#### Screenshot

https://github.com/user-attachments/assets/be59a1be-ec90-44f9-bffb-4e3ac034353d

**Comment by ramezsaeed** [comment]
Verified:
- Mainnet
- 2.7.0
- deploy a vm on a node in Poland and ssh is working fine.

![Image](https://github.com/user-attachments/assets/11d498f9-bdfd-4549-b54d-f818764c404a)
![Image](https://github.com/user-attachments/assets/0effdec2-a8ed-406b-ace3-517e3006593c)
![Image](https://github.com/user-attachments/assets/7d9918a7-8078-462e-9177-06300519c2b1)
